src/Core/Utility/VersionUtility.cs

```diff
@@ -664,6 +678,27 @@ public static string GetShortFrameworkName(FrameworkName frameworkName)
                 // only show version part if it's > 0.0.0.0
                 if (frameworkName.Version > new Version())
                 {
+                    if (frameworkName.Identifier.Equals(NetStandardAppFrameworkIdentifier, StringComparison.OrdinalIgnoreCase)
+                        || frameworkName.Identifier.Equals(NetStandardFrameworkIdentifier, StringComparison.OrdinalIgnoreCase)
+                        || frameworkName.Identifier.Equals(NetPlatformFrameworkIdentifier, StringComparison.OrdinalIgnoreCase))
+                    {
+                        if (frameworkName.Version.Major > 9

```

Comment:
Shouldn't it just always call GetDecimalVersionString here? Or is the format different?